config/configurations.py

```diff
@@ -184,62 +118,35 @@ def get_setting(path, section, setting):
     return value
 
 
-def update_setting(path, section, setting, value):
-    """Update a setting in the INI file.
+def update_setting(path: str, section: str, setting: str, value: Any):
+    """Update a setting in the TOML file.
```

Comment:
Might want to look into handling updating multiple settings as every call to this function will result in unnecessary IO hits for each setting.